lib/mongo/query_cache.rb

```diff
@@ -83,6 +83,41 @@ def cache_table
       def clear_cache
         Thread.current["[mongo]:query_cache"] = nil
       end
+
+      def write(cursor, options = {})
```

Comment:
Are these public API?

Reply:
They are currently, but they don't have to be! Do you think it would be better to make them private?

Reply:
I think the methods that take/return cursors should default to private.